sklearnex/linear_model/logistic_regression.py

```diff
@@ -132,6 +132,8 @@ def predict(self, X):
                 X,
             )
 
+        predict = wrap_output_data(_predict)
+
```

Comment:
Can you elaborate on what impact this has? just curious

Reply:
the score method needs to access numpy array versions of data in order to properly calculate using sklearn. accessing the result of predict without having turning it into a dpctl/dpnp tensor is required, which means having access to predict without wrap_output_data. I make a new method called _predict which then can be used in score, and then expose predict by directly wrapping with wrap_output_data.